org.openhab.binding.zigbee/README.md

```diff
@@ -18,6 +18,18 @@ demo.things:
 Thing zigbee:coordinator_cc2531:stick1 "Zigbee USB Stick" [zigbee_port="/dev/ttyACM0", zigbee_baud="38400"]
 ```
 
+#### Serial port Configuration
+
+If you are using a config file, the serial port needs to be configured as shown in the above example. Below is a list of the relevat settings concerning the serial port.
+
+| Setting               | Allowed values  |
+| --------------------- | --------------- |
+| `zigbee_baud`         | `38400`, `57600`, `115200` |
+| `zigbee_flowcontrol ` | `0` = None, `1` = Hardware (CTS/RTS), `2` = Software (XOn/XOff) |
+| `zigbee_port`         | The name of the serial port, e.g. `/dev/ttyUSB1` or `/dev/ttyZigbee` or whatever the port is named on your system |
+
+Note that these serial port configuration values should be provided as strings in the config file, that is, inside double quotes, as shown in the example above.
```

Comment:
I'm surprised that this is the case - the baud rate is defined as an integer in the configuration file, so it should not contain the quotes.

Reply:
You are right. It works with and without quotes, so it seems. I'll remove that note.